dev/io.openliberty.microprofile.reactive.messaging.internal_fat/fat/src/io/openliberty/microprofile/reactive/messaging/fat/telemetry/ReactiveMessagingTelemetryTestWithJAXRS.java

```diff
@@ -0,0 +1,128 @@
+public class ReactiveMessagingTelemetryTestWithJAXRS extends FATServletClient {
+
+    private static final String APP_NAME = "ReactiveMessagingTelemetryApp";
+    public static final String SERVER_NAME = "RxMessagingServerWithTelemetry";
+    private static final String TOPIC_NAME = "JAXRSTopic";
+
+    @Server(SERVER_NAME)
+    public static LibertyServer server;
+
+    @ClassRule
+    public static RepeatTests r = ReactiveMessagingActions.repeat(SERVER_NAME,
+                                                                  ReactiveMessagingActions.MP61_RM30,
+                                                                  ReactiveMessagingActions.MP50_RM30,
+                                                                  ReactiveMessagingActions.MP60_RM30);
+
+    @BeforeClass
+    public static void setup() throws Exception {
+
+        PropertiesAsset appConfig = new PropertiesAsset()
+                        .addProperty(AbstractKafkaTestServlet.KAFKA_BOOTSTRAP_PROPERTY, KafkaTests.kafkaContainer.getBootstrapServers())
+                        .include(ConnectorProperties.simpleIncomingChannel(KafkaTests.connectionProperties(), ConnectorProperties.DEFAULT_CONNECTOR_ID,
+                                                                           RmTelemetryProcessingBean.CHANNEL_IN, "ReactiveMessagingTelemetryApp", TOPIC_NAME))
+                        .include(ConnectorProperties.simpleOutgoingChannel(KafkaTests.connectionProperties(), ConnectorProperties.DEFAULT_CONNECTOR_ID,
+                                                                           ReactiveMessagingResource.EMITTER_CHANNEL, TOPIC_NAME))
+                        .addProperty(CONNECTION_PROPERTIES_KEY, KafkaTestClientProvider.encodeProperties(KafkaTests.connectionProperties()));
+
+        WebArchive war = ShrinkWrap.create(WebArchive.class, APP_NAME + ".war")
+                        .addAsLibraries(kafkaClientLibs())
+                        .addAsManifestResource(kafkaPermissions(), "permissions.xml")
+                        .addClasses(ReactiveMessagingApplication.class, ReactiveMessagingResource.class, RmTelemetryProcessingBean.class, RmTelemetryReceptionBean.class)
+                        .addPackage(KafkaTestConstants.class.getPackage())
+                        .addPackage(AbstractKafkaTestServlet.class.getPackage())
+                        .addAsResource(new StringAsset("otel.sdk.disabled=false"), "META-INF/microprofile-config.properties")
+                        .addAsResource(appConfig, "META-INF/microprofile-config.properties");
+
+        ShrinkHelper.exportAppToServer(server, war, SERVER_ONLY);
+        server.startServer();
+    }
+
+    @AfterClass
+    public static void teardown() throws Exception {
+        try {
+            server.stopServer();
+        } finally {
+            KafkaUtils.deleteKafkaTopics(KafkaTests.getAdminClient());
+        }
+
+    }
+
+    @Test
+    public void invokeEmitterViaREST() throws Exception {
```

Comment:
For this, I would like us to send multiple messages that go through the chain, this is so we know nothing locks the system, given we have seen in some scenarios that we can send messages, but reading them back causes issues.

https://github.com/OpenLiberty/open-liberty/blob/9d36be21a7404f673724f48b8ada0a1808fcf1d7/dev/io.openliberty.microprofile.reactive.messaging.internal_fat/fat/src/io/openliberty/microprofile/reactive/messaging/fat/kafka/emitter/restful/KafkaEmitterRestfulTest.java#L107-L130

Covers the sending of multiple messages and asserting on them.